### PR TITLE
Obsolete cockpit-dashboard

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

File: images/scripts/centos-7.install [hunk 1]
```diff
@@ -3,6 +3,6 @@
 set -e
 
 # remove cockpit distro packages, testing with upstream master
-rpm --erase --verbose cockpit cockpit-ws cockpit-bridge cockpit-system cockpit-dashboard
+rpm --erase --verbose cockpit cockpit-ws cockpit-bridge cockpit-system
```
Comment (martinpitt) [inline images/scripts/centos-7.install:6]: I'd keep that -- I assume you don't want to drop c-dashboard from the cockpit rhel-7.9 branch.

File: images/scripts/fedora-coreos.install [hunk 2]
```diff
@@ -4,15 +4,15 @@ set -eux
 # install/upgrade RPMs that apply to Fedora CoreOS
 # Note: cockpit-selinux would be desirable, but needs setroubleshoot-server which isn't installed
 cd /var/tmp/build-results/
-rpm-ostree install cockpit-bridge-*.rpm cockpit-dashboard-*.rpm \
+rpm-ostree install cockpit-bridge-*.rpm \
```
Comment (martinpitt) [inline images/scripts/fedora-coreos.install:7]: This causes the cockpit/fedora-coreos failures. I assume this PR needs to land in lockstep with the cockpit PR that removes it?

File: images/scripts/rhel-7-9.install [hunk 3]
```diff
@@ -3,6 +3,6 @@
 set -e
 
 # remove cockpit distro packages, testing with upstream master
-rpm --erase --verbose cockpit cockpit-ws cockpit-bridge cockpit-system cockpit-dashboard
+rpm --erase --verbose cockpit cockpit-ws cockpit-bridge cockpit-system
```
Comment (martinpitt) [inline images/scripts/rhel-7-9.install:6]: Dito, let's keep 7-9 as-is.

File: images/scripts/rhel.setup [hunk 4]
```diff
@@ -300,7 +300,7 @@ fi
 # Pre-install cockpit packages from base, to check for API breakages
 # and more convenient interactive debugging
 if [ "${IMAGE#rhel-7}" != "$IMAGE" ] || [ "${IMAGE#centos-7}" != "$IMAGE" ] ; then
-    $YUM_INSTALL cockpit cockpit-dashboard
+    $YUM_INSTALL cockpit
```
Comment (martinpitt) [inline images/scripts/rhel.setup:303]: Dito

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: images/scripts/fedora-coreos.install (hunk 2 above)
Comment (marusak) [reply images/scripts/fedora-coreos.install:7]: Yeah, here I checked that all works as expected:
https://logs.cockpit-project.org/logs/pull-14905-20201130-193223-77c80724-fedora-coreos-bots-1423/log.html

**Comment by marusak** [comment]
Updated

File: images/scripts/lib/atomic.install [hunk 5]
```diff
@@ -168,7 +168,7 @@ class AtomicCockpitInstaller:
         """ Install the latest cockpit RPMs in our container"""
         rpm_args = []
         for package in self.rpms:
-            if 'cockpit-ws' in package or 'cockpit-dashboard' in package or 'cockpit-bridge' in package:
+            if 'cockpit-ws' in package or 'cockpit-bridge' in package:
```
Comment (martinpitt) [inline images/scripts/lib/atomic.install:171]: rhel-atomic is also RHEL 7, let's not touch this.

File: images/scripts/rhel.setup [hunk 6]
```diff
@@ -103,7 +103,7 @@ EOF
 
     elif [ "${IMAGE#rhel-8*}" != "$IMAGE" ]; then
         case "$IMAGE" in
-            rhel-8-3) REPO="rhel-8/nightly/RHEL-8/latest-RHEL-8.3" ;;
+            rhel-8-3) REPO="rhel-8/nightly/updates/RHEL-8/latest-RHEL-8.3.1" ;;
```
Comment (martinpitt) [inline images/scripts/rhel.setup:106]: While this change makes sense, let's leave that to PR #1339

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: images/scripts/rhel.setup (hunk 6 above)
Comment (marusak) [reply images/scripts/rhel.setup:106]: oh, damn, I had some local changes I committed by accident.

File: images/scripts/lib/atomic.install (hunk 5 above)
Comment (marusak) [reply images/scripts/lib/atomic.install:171]: rhel-atomic or atomic? (you commented on atomic)

Comment (martinpitt) [reply images/scripts/lib/atomic.install:171]: rhel-atomic.install is the only remaining OS install script that uses this file. So they are really synonyms these days.

**Review by martinpitt — APPROVED** [review]
Thanks! Please adjust the commit message a bit on merging: Maybe "images: Stop installing cockpit-dashboard on fedora-coreos", and giving a reference to the cockpit PR in the long description?

**Pull request merged**